frontend/src/components/ReportSelect/index.tsx

```diff
@@ -58,140 +67,149 @@ export const ReportSelect = ({ data }: { data: IData }) => {
-              <FormControlLabel
-                control={<Checkbox />}
-                label={checkbox.value}
-                key={checkbox.value}
+              <FormInput
+                setFieldValue={setFieldValue}
+                fieldValue={values?.[fieldName] || ""}
+                fieldName={fieldName}
+                fieldType={fieldType}
+                fieldLabel={fieldLabel}
+                handleChange={handleChange}
+                width={width}
+                key={fieldName}
+                tableName={tableName}
+                pickerName={pickerName}
               />
             );
-          })}
-        </FormGroup>
-      </FormControl>
-    );
+          }
+        }
+      );
+    }
   };
 
-  const renderParameters = (
-    reportParameters: {
-      name: string;
-      formLabel: string;
-      components: Array<ISelect | IDate | ICheckbox>;
-    },
-    state: string
-  ) => {
-    return reportParameters.components.map((component: ISelect | IDate | ICheckbox) => {
-      if (component.parents.includes(state)) {
-        switch (component.input) {
-          case "select":
-            return renderSelect(component);
-          case "date":
-            return renderDate(component);
-          case "checkbox":
-            return renderCheckbox(component);
-          default:
-            return <FormControl></FormControl>;
-        }
+  const renderDescription = () => {
+    return reportDescription.options.map((description) => {
+      if (description?.reportType?.includes(currentReportType as unknown as string, 0)) {
+        return <p key={description.value}>{description.value}</p>;
       }
     });
   };
 
-  const renderDescription = (reportDescription: IDescription, state: string) => {
-    return reportDescription.options.map(
-      (description: { id: number; value: string; parent: string }) => {
-        if (state === description.parent) {
-          return <p key={description.id}>{description.value}</p>;
-        }
-      }
-    );
+  const onExportButtonClick = (values: {
+    [key: string]: { value: number | string; label: string };
+  }) => {
+    const baseUri = `https://localhost:8080/report/projects`;
+    const reportUri = `${values.report_type}`;
+    // This will need to change as more report types are added
+    const url = `${baseUri}/${values.project.value}/${reportUri}`;
+    axios(url, {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+        Accept: "application/json",
+        responseType: "arraybuffer",
+      },
+      responseType: "blob",
+    })
+      .then((response) => {
+        const fileURL = window.URL.createObjectURL(response.data);
+        const alink = document.createElement("a");
+        alink.href = fileURL;
+        alink.download = `${values.report_type}.pdf`;
+        alink.click();
+      })
+      .catch((err) => {
+        // Handle the error gracefully - to be addressed in a future ticket
+        alert(err);
```

Comment:
not a fan of alerts, but i think i know where you are coming from by using this.

Reply:
I agree, this was a simple stopgap measure until we decide how to handle this.